src/Storefront/Theme/ThemeMergedConfigBuilder.php

```diff
@@ -121,6 +123,24 @@ public function getPlainThemeConfiguration(string $themeId, Context $context): a
             $themeConfig['themeTechnicalName'] = $theme->getTechnicalName();
         }
 
+        $themeConfig['inheritedThemeTechnicalNames'] = [];
```

Comment:
Instead of building a new inheritance order for the technical names, we should consider using the `configInheritance` setting of the theme, which already includes the desired order of technical names for the config inheritance. As a developer I would expect that the snippets follow the same inheritance structure I have defined in my `configInheritance` setting.